# CustomService.srv
# Request fields
uint32 arg_1
---
# Response fields
string out_1

